moviebot/explainability/explainable_user_model_tag_based.py

```diff
@@ -1,9 +1,13 @@
-"""This module contains the ExplainableUserModelTagBased class.
+"""Class for creating a tag-based user model explanations.
 
-The class generates explanations for user preferences in the movie domain based
-on templates loaded from a YAML file.
+The class generates explanations for user preferences in the movie domain.
+Currently, the explanations are based on the explicit movie tags/attributes.
```

Comment:
Nit: "... are based on movie tags/attributes that were explicitly mentioned by the user in the conversation."

Reply:
Done